core/cloudflow-operator/src/main/scala/cloudflow/operator/Operator.scala

```diff
@@ -98,6 +99,7 @@ object Operator {
 
   def handleConfigurationInput(
       client: KubernetesClient,
+      runners: Map[String, Runner[_]],
```

Comment:
I don't understand, from the current diff, what we are getting by passing around the runners, am I missing some context?

Reply:
yes, errorAction